setup/setup.iss

```diff
@@ -7,11 +7,14 @@
 [Setup]
 AppId={{BDD60DE7-9374-463C-8E74-8227EB03E28F}
 AppName=Dash Faction
+AppVerName=Dash Faction
 AppVersion={#AppVer}
 AppPublisher=rafalh
-AppPublisherURL=https://ravin.tk/
-AppSupportURL=https://ravin.tk/
-AppUpdatesURL=https://ravin.tk/
+AppPublisherURL=https://rafalh.dev/
+AppSupportURL=https://rafalh.dev/
+AppUpdatesURL=https://rafalh.dev/
+SetupIconFile="..\launcher\res\DF.ico"

```

Comment:
I don't like the idea of using DashFaction main icon for the setup program. IMHO it would be a little confusing. I prefer the default icon that clearly indicates that the setup program is just what it is and not the real app.